### DISCUSSION TITLE
Network Policies

### DISCUSSION THREAD

**Discussion by hans-fischer** [Mailu/Mailu#3237] [User support] [answered]
Hello Together,

at first I want to say that you guys doing a realy great job. This Mail-Server-Stack is awesome! I want to migrate from my current VM-based stack to this one but at first I want to automate everything in ansible and establish the high security standards of my k8s cluster.

I try to develop network policies of the mailu-stack. But I struggle to find documentation who connects to which container.

Could one for example tell me where I can find the roundcube-config that points out which endpoints are configured?

Thank you all for your help.
Hans

**Comment by nextgens** [comment, marked as answer]
https://github.com/Mailu/Mailu/blob/master/docs/mailu-network-diagram.ipynb

Keep in mind that this is different on master and 2.0 though

**Comment by hans-fischer** [comment]
That is Network-Policy Gold.
Thank you very much!
